tests/row_generation/test_Helpers.py

```diff
@@ -71,6 +72,19 @@ def test_convert_pn(self):
             with self.subTest(input_pn=input_pn, expected_output_pn=expected_output_pn):
                 self.assertEqual(convert_pn(input_pn), expected_output_pn)
 
+class HelpersGenerateStartingRowTests(unittest.TestCase):
+    def test_generate_without_custom_row(self):
+        self.assertListEqual(Row([Bell(0), Bell(1), Bell(2), Bell(3)]), generate_starting_row(4))
+
+    def test_generate_with_custom_row(self):
+        self.assertListEqual(Row([Bell(3), Bell(2), Bell(1), Bell(0)]), generate_starting_row(4, "4321"))
+        self.assertListEqual(Row([Bell(3), Bell(2), Bell(1), Bell(0)]), generate_starting_row(4, "432"))
+        self.assertListEqual(Row([Bell(3), Bell(1), Bell(0), Bell(2)]), generate_starting_row(4, "42"))
+        self.assertListEqual(Row([Bell(1), Bell(0), Bell(2), Bell(3)]), generate_starting_row(4, "2"))
```

Comment:
Shouldn't these tests break if the row parsing throws an error on incorrect rows?